firebase-inappmessaging-display/src/main/java/com/google/firebase/inappmessaging/display/FirebaseInAppMessagingDisplayRegistrar.java

```diff
@@ -46,7 +45,6 @@
     return Arrays.asList(
         Component.builder(FirebaseInAppMessagingDisplay.class)
             .add(Dependency.required(FirebaseApp.class))
-            .add(Dependency.required(AnalyticsConnector.class))
```

Comment:
I think you should also update line 59 below to something like

```java
FirebaseApp firebaseApp = container.get(FirebaseApp.class);
```

Reply:
Done